lib/bookmark_web/controllers/archive_controller.ex

```diff
@@ -23,15 +23,15 @@ defmodule BookmarkWeb.ArchiveController do
   def show(conn, %{"id" => id}) do
     user = conn.assigns.current_user
 
-    balance = Bookmark.Wallets.balance(user.wallet_key)
+    balance = Bookmark.Wallets.balance(user)
```

Comment:
I'm just curious as to the reason for this change. I know the `balance` takes a user or a key, I would just like to know why you are sending a user here instead of the key

Reply:
@green-john it is because to cover the `user = nil` that is where is a client using without a user